github-dork.py

```diff
@@ -35,12 +36,43 @@ def search_wrapper(gen):
         except Exception as e:
             raise e
 
-def search(repo_to_search=None, user_to_search=None, gh_dorks_file=None):
+def metasearch(repo_to_search=None, user_to_search=None, gh_dorks_file=None, active_monit=None):
+    if active_monit is None:
+	search(
+        	repo_to_search,
+        	user_to_search,
+        	gh_dorks_file,
+        	active_monit
+	)
+    else:
+	monit(  gh_dorks_file,
+		active_monit
+	)	
+
+def monit(gh_dorks_file=None,active_monit=None):
+    if gh_user is None:
+	raise Exception('Error, env Github user variable needed')
+    else:
+	print "Monitoring user private feed searching new code to be 'dorked'. Every new merged pull request trigger user scan."
```

Comment:
@ffr4nz Could you put these in a brace such as `print("Monitoring...")`? Its in bunch of places. This ensures it does not break on python 3